crime-applications-adaptor/src/test/resources/data/crimeapply/application_2.json

```diff
@@ -0,0 +1,37 @@
+{
+  "id": "7ce78426-93dc-437f-a5fe-7203dcbf103e",
+  "schema_version": 1.0,
+  "reference": 6000309,
```

Comment:
Can you rename these JSON files to make it clearer what they represent, in particular they are JSON representations of `MaatApplication` objects and have a unique usn/ref which can be used to identify them 

i.e. application_2.json -> MaatApplication_ 6000309.json

Reply:
New filenames look good